apps/myaccount/src/components/multi-factor-authentication/authenticators/fido-authenticator.tsx

```diff
@@ -91,22 +91,21 @@ export const FIDOAuthenticator: React.FunctionComponent<FIDOAuthenticatorProps>
         // Exception is not handled here since this request is dispatched on page load.
         // TODO: Add a logger to print the exception
         startFidoUsernamelessFlow()
-            .then((response) => {
+            .then((response: any) => {
                 setFidoFlowStartResponse(response);
             });
 
         // Exception is not handled here since this request is dispatched on page load.
         // TODO: Add a logger to print the exception
         commonConfig.accountSecurityPage.mfa.fido2.allowLegacyKeyRegistration &&
             startFidoFlow()
-                .then((response) => {
+                .then((response: any) => {
                     setOldFidoFlowStartResponse(response);
                 });
-
     }, []);
 
     const activeForm: string = useSelector((state: AppState) => state.global.activeForm);
-    const dispatch = useDispatch();
+    const dispatch: Dispatch<any> = useDispatch();
```

Comment:
Can we use the dispatch in redux so that we can avoid the any data type?

Reply:
Fixed